extension/vector/src/function/query_hnsw_index.cpp

```diff
@@ -96,7 +96,7 @@ static std::unique_ptr<TableFuncBindData> bindFunc(main::ClientContext* context,
         LogicalTypeID::ARRAY);
     KU_UNUSED(auxInfo);
 
-    auto returnColumnNames = std::vector<std::string>{"nn", "_distance"};
+    auto returnColumnNames = std::vector<std::string>{"nn", "distance"};
```

Comment:
I suggest defining the return column names as two constexprs in the queryHNSWFunction body.